target/linux/pistachio/base-files/etc/init.d/bt-config

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+START=85
+STOP=85
+
+config_bluetooth() {
+	. /lib/functions/mac.sh
+	local IS_BLUETOOTHD_INITIALIZED
+	while [ -z "$IS_BLUETOOTHD_INITIALIZED" ]
+	do
+		IS_BLUETOOTHD_INITIALIZED=`ps | grep bluetoothd | grep -v grep`
+	done
+	# give some time for the bluetoothd to come up before applying 
+	# configuration
+	sleep 3
```

Comment:
The bluetoothd takes approx. 2-3 secs to find the controller, this is the reason this script will be executed last in the init.d